articles/batch/batch-sig-images.md

```diff
@@ -45,7 +45,7 @@ Using a Shared Image configured for your scenario can provide several advantages
 >
 > The image can be in a different region as long as it has replicas in the same region as your Batch account.
 
-If you use a Microsoft Entra application to create a custom image pool with an Azure Compute Gallery image, that application must have been granted an [Azure built-in role](../role-based-access-control/rbac-and-directory-admin-roles.md#azure-roles) that gives it access to the Shared Image. You can grant this access in the Azure portal by navigating to the Shared Image, selecting **Access control (IAM)** and adding a role assignment for the application.
+If you use a Microsoft Entra application to create a custom image pool with an Azure Compute Gallery image, that application must have been granted an [Azure built-in role](../role-based-access-control/rbac-and-directory-admin-roles.md#azure-roles) that gives it access to the Shared Image. You can grant this access in the Azure portal by navigating to the Shared Image, selecting **Access control (IAM)** and adding a role assignment for the application. Please note that Reader permissions on the Azure Compute Gallery image is not enough as it is required minimum the following action 'Microsoft.Compute/disks/beginGetAccess/action'.
```

Comment:
```suggestion
If you use a Microsoft Entra application to create a custom image pool with an Azure Compute Gallery image, that application must have been granted an [Azure built-in role](../role-based-access-control/rbac-and-directory-admin-roles.md#azure-roles) that gives it access to the Shared Image. You can grant this access in the Azure portal by navigating to the Shared Image, selecting **Access control (IAM)** and adding a role assignment for the application. Keep in mind that the Reader role does not provide adequate permissions because access requires performing the following action:  `Microsoft.Compute/disks/beginGetAccess/action`.
```
The sentence as written is confusing and doesn't follow Microsoft style. I've made a suggestion, but

Reply:
Please pull this sentence into "NOTE", we are flagging customer on the requirement as below:
*NOTE:* Please note that the Reader permissions for the Azure Compute Gallery image are inadequate as they necessitate the execution of the following minimum action: Microsoft.Compute/disks/beginGetAccess/action for appropriate access.